cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -87,6 +87,19 @@ func (p *Provisioner) ProvisionHostPath(ctx context.Context, opts pvController.P
 		return nil, pvController.ProvisioningFinished, iErr
 	}
 
+	bsoft := opts.StorageClass.Parameters["bsoft"]
+	bhard := opts.StorageClass.Parameters["bhard"]
+
+	if bsoft != "" ||
+		bhard != "" {
```

Comment:
most of the possible validations are now covered

Reply:
we can also use decimal values now

Reply:
> Is the code formatted??

Is this not correct format?
In my system there is small gap but after pushing it becomes big , it may be related to `Tab` spaces

Reply:
> we can also use decimal values now

No, we should not use decimals as there is a limitation for decimal values while applying quotas.` The limits cannot be in decimals`.  That's what the check needs to be- check whether the limits are not given in decimals because that would cause errors while enforcing quotas.

Reply:
> Is this not correct format?
In my system there is small gap but after pushing it becomes big , it may be related to Tab spaces or it auto formats beacuse it can only be seen in if statements

Just run `make fmt` command from the root of the project folder.